### DISCUSSION TITLE
HM-Sec-SC keine Statusänderungen mehr

### DISCUSSION THREAD

**Discussion by FrankyB** [OpenCCU/OpenCCU#2072] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Alle meine HM-Sec-SC geben schon länger keinen Status mehr an Raspberrymatic - der Magnet geht an den Sensor, der Sensor wird gelb.

Die LED zeigt nur noch "orange" -ich meine als ich die Dinger installiert hatte war das rot/grün im Wechsel

Der Status des Sensors in der Zentrale bleibt allerdings gleich -keine Änderung.

### Describe the behavior you expected

Der Status sollte sich von "geschlossen" auf "offen" ändern, die LED hatte meiner Meinung nach auch vorher anders reagiert.

### Steps to reproduce the issue

- HM-Sec-SC anlernen
- Magnet hinhalten => Status geschlossen
- Magnet weg => Status geschlossen (also keine Änderung)

### What is the version this bug report is based on?

3.65.8.20220831-rpi3

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

HM-MOD-RPI-PCB

### Anything in the logs that might be useful for us?

```txt
`
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C> @650283: #158 LLMAC RX @20779ms -45dBm 00 A0 41 20 2B BF 1F 7B FC 01 01 C8
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: Bidcos RX: #00[BiDi|Ren] 202BBF->1F7BFC SwitchConditional: 01 01 C8
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: GetAckActionForIncomingTelegram() channel=1
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: GetAckActionForIncomingTelegram() ackAction=AckAction_AesChallenge, wakeup=None
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: BidcosMacController::SendAESChallenge()
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: MacController::OnDownstreamFrame(#255 LLMAC TX @20886ms [10k,NoCCA] 00 A0 02 1F 7B FC 20 2B BF 04 A4 7F A1 22 5E 95 02)
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: BidcosMacController::SendAESChallenge() - autoTxState=AutoTxState_AesChallenge, txState=TxState_WaitCoproResponseAesSolution
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: AutoTx: #00[BiDi|Ren] 1F7BFC->202BBF Ack: 04 A4 7F A1 22 5E 95 02
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C<: #240 LLMAC TX @20886ms [10k,NoCCA] 00 A0 02 1F 7B FC 20 2B BF 04 A4 7F A1 22 5E 95 02
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C< @650285: bin:FD 00 17 03 F0 06 51 96 80 00 A0 02 1F 7B FC 7C 20 2B BF 04 A4 7F A1 22 5E 95 02 F5 F6
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C> @650412: #240 LLMAC Response ACK @20887: 51 97
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: _txState = TxState_WaitAesSolution
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C> @650555: #159 LLMAC RX @21051ms -43dBm 00 A0 03 20 2B BF 1F 7B FC E7 13 04 F6 7E 83 7D FA 13 B3 AA 13 9F 5F 69 FE
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: Bidcos RX: #00[BiDi|Ren] 202BBF->1F7BFC AesSolution: E7 13 04 F6 7E 83 7D FA 13 B3 AA 13 9F 5F 69 FE
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: GetAckActionForIncomingTelegram() ackAction=AckAction_AesAck, wakeup=None
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: _txState = TxState_Idle
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: AES solution not accepted
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: MacController::OnDownstreamFrame(#255 LLMAC TX @21148ms [10k,NoCCA] 00 80 02 1F 7B FC 20 2B BF 00 00 00 00 00)
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: AutoTx: #00[Ren] 1F7BFC->202BBF Ack: 00 00 00 00 00
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C<: #241 LLMAC TX @21148ms [10k,NoCCA] 00 80 02 1F 7B FC 20 2B BF 00 00 00 00 00
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: A<: #52 HmBidcos RxTelegram AuthFailed #00[BiDi|Ren] 202BBF->1F7BFC SwitchConditional: 01 01 C8
Nov 26 16:40:17 homematic-ccu2 user.debug multimac: C< @650556: bin:FD 00 14 03 F1 06 52 9C 80 00 80 02 1F 7B FC 7C 20 2B BF 00 00 00 00 00 5A 4D
Nov 26 16:40:17 homematic-ccu2 user.debug rfd: RX for KEQ0027886: @3029906888 RSSI=-45dB 0x202BBF -> 0x1F7BFC CONDITIONAL_SWITCH [OEQ0302766]:   CNT=0,RPTEN=1,RPTED=0,BIDI=1,BURST=0,WAKEUP=0,WAKEMEUP=0,BCAST=0,TYPE=0x41   COUNTER = 1   CHANNEL = 1   LOWBAT = 0   DURATION = 0   CONDITION = 200   TIMES = 0
Nov 26 16:40:17 homematic-ccu2 user.debug rfd: Event: KEQ0027886:1.LOWBAT=false
`


bzw wird bei Magnet abziehen daraus ein 
`
Nov 26 16:42:57 homematic-ccu2 user.debug rfd: RX for KEQ0027886: @3030066334 RSSI=-51dB 0x202BBF -> 0x1F7BFC INFO_ACTUATOR_STATUS [OEQ0302766]:   CNT=1,RPTEN=1,RPTED=0,BIDI=1,BURST=0,WAKEUP=0,WAKEMEUP=1,BCAST=1,TYPE=0x10   CHANNEL = 1   STATUS = 200   STATE = 0   CLOCK = 0   LOWBAT = 0   DUTY_CYCLE = 0   RSSI = 0`
```

### Additional information

Wie gesagt, ich weiß nicht seit wann das nicht mehr geht, aber sicher schon eine ganze Weile. Ich habe gerade noch einen HmIP-SWDM angelernt & der funktioniert wie er soll. Leider hab ich einige der HM-Sec-SD verbaut, und die scheinen alle den Geist aufgegeben zu haben. Ich habe gerade noch neue Batterien eingebaut, doch an dem Status in der Zentrale ändert sich nichts.

**Comment by FrankyB** [comment, marked as answer]
Sorry, ich hab die Kommunikation von "gesichert" auf "standard" gesetzt,jetzt gehen die Dinger....

**Reply by jp112sdl** [reply]
> Nov 26 16:40:17 homematic-ccu2 user.debug multimac: AES solution not accepted

Hast du mal einen Sicherheitsschlüssel gesetzt/geändert, der nicht an alle Geräte übertragen wurde?

**Reply by FrankyB** [reply]
Kann gut sein, wie geschrieben, das muss ja schon ewig her sein. Ich hab gerade mal zum Test die Übertragungsart wieder auf "gesichert" umgestellt & der Status wird wieder nicht übertragen. Ein anderer Kontakt geht aber - einer der näher an der Zentrale steht. Du scheinst recht zu haben. Nur so aus Neugier, wie kann ich den aktuellen Schlüssel an den Sensor übertragen?

**Reply by jp112sdl** [reply]
Da das Challenge-Response-Auth aktuell fehlschlägt, kann auch kein neuer Schlüssel übertragen werden. 
Das kannst du nur durch "Ablernen mit Werksreset" und erneutem Anlernen erreichen.
Und wirklich gut drauf achten, dass **"mit Werksreset"** abgelernt wird.

**Reply by FrankyB** [reply]
Ok, das lass ich lieber, sonst zerschießts mir meine zusammengeklickten Wenn...dann Programme. Danke für die Info!

**Comment by jens-maus** [comment]
Tut mir leid, aber das kann kein genereller Bug in RaspberryMatic sein, sondern wir eine lokales Problem sein. Deshalb konvertierte ich das mal in einen Diskussionsbeitrag und vllt melden sich ja andere die das gleiche Problem haben/hatten.

**Reply by FrankyB** [reply]
Danke! Mit der Lösung die Übertragungsart auf "Standard" (unverschlüsselt) einzustellen kann ich an dem Türkontakt der mir die Heizung zudreht leben. Wäre nur schade gewesen wenn ich die Sensoren verschrotten hätte müssen.
